tests/test_discrete_problem.py

```diff
@@ -41,7 +41,7 @@ def test_get_solve_discrete_problem_illustrative():
         ],
     )
 
-    got = solve_discrete_problem(cc_values, params=None)
+    got = solve_discrete_problem(cc_values, params={"ignore": "this"})
```

Comment:
Use an empty dict or explain. 
```suggestion
    got = solve_discrete_problem(cc_values, params={"ignore": "because ..."})
```